### PR TITLE
Feature: MathJax support

### PR DESCRIPTION
**Pull request by rabarar**
Add script for MathJax support

### CODE REVIEW

**Comment by rabarar** [comment]
That makes sense. Although I'm not sure how to do that :)

I'll take a look and see if i can figure it out.

Rob

**Comment by rabarar** [comment]
PR - mathjax support

**Comment by rabarar** [comment]
Got it, resubmitted the PR...

**Comment by ChrisStayte** [comment]
Do we want the example site 'toml' to have this in there as an option for future users to know it exists?

**Comment by rabarar** [comment]
Ah, I should have modified the example config file. Great idea.

**Comment by Lednerb** [comment]
@ChrisStayte yes, that would be necessary.

@rabarar  A hint in the documentation under the _Features_ section "MathJax Support" would be helpful, too.

**Comment by rabarar** [comment]
No problem. I'll add it later this morning.

File: README.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Note that on mobile devices the navigation will still be collapsed because other
 
 ### MathJAX Markup
 
-If you want to add [MathJAX](https://github.com/rabarar/bilberry-hugo-theme.git) markup support, set parameter `enable_mathjax` option to `true in our site's config file.
+If you want to add [MathJAX](https://github.com/rabarar/bilberry-hugo-theme.git) markup support, set parameter `enable_mathjax` option to `true` in our site's config file.
```
Comment (Lednerb) [inline README.md]: Please set the MathJAX link to the project website.

File: layouts/partials/mathjax.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<!-- Include MathJax library -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```
Comment (Lednerb) [inline layouts/partials/mathjax.html]: Please include the `integrity` hash and the `crossorigin` attribute from cloudflare to the `script` tag for better security.

**Comment by rabarar** [comment]
> On Nov 21, 2017, at 4:00 PM, Sascha Brendel <notifications@github.com> wrote:
> 
> @Lednerb requested changes on this pull request.
> 
> Thx for your contribution!
> 2 Little things before we can merge this.
> 
> In README.md <https://github.com/Lednerb/bilberry-hugo-theme/pull/34#discussion_r152399104>:
> 
> > @@ -158,7 +158,7 @@ Note that on mobile devices the navigation will still be collapsed because other
>  
>  ### MathJAX Markup
>  
> -If you want to add [MathJAX](https://github.com/rabarar/bilberry-hugo-theme.git) markup support, set parameter `enable_mathjax` option to `true in our site's config file.
> +If you want to add [MathJAX](https://github.com/rabarar/bilberry-hugo-theme.git) markup support, set parameter `enable_mathjax` option to `true` in our site's config file.
> Please set the MathJAX link to the project website.
> 
> 

ah, cut and paste error! thanks for catching that…

> 
> In layouts/partials/mathjax.html <https://github.com/Lednerb/bilberry-hugo-theme/pull/34#discussion_r152399742>:
> 
> > @@ -0,0 +1,8 @@
> +
> +<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
> Please include the integrity hash and the crossorigin attribute from cloudflare to the script tag for better security.
> 

I’m assuming you want something like this? i’m not a js jockey so correct me if this isn’t correct:


<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
integrity="sha256-48344fd55558bbeb600062a175d052979f9ece87c7299788f8ecf16a46c87bf6"
crossorigin="anonymous"
</script>

File: layouts/partials/mathjax.html (hunk 2 above)
Comment (Lednerb) [reply layouts/partials/mathjax.html]: @Lednerb, does not work as expected via cloudflare cdnjs. Everything ok.

**Comment by Lednerb** [comment]
I've updated and integrated the MathJax support to the master branch.

Thx for your contribution!

**Pull request merged**